mvlearn/datasets/base.py

```diff
@@ -90,6 +98,14 @@ def load_UCImultifeature(select_labeled="all", shuffle=False,
         raise ValueError("If selecting examples by label, must select "
                          "at least 1 and no more than 10.")
 
+    views = list(dict.fromkeys(views))
+    if len(views) == 0:
+        raise ValueError("If views, must select at least 1.")
```

Comment:
Just for better wording, I'd say "if choosing specific views, must select at least 1". What do you think?

Reply:
Yeah, I like that wording too. I'll change it to that.